includes/tutorials-export-data-to-excel.md

```diff
@@ -0,0 +1,217 @@
+{WidgetName} allows users to export grid data as Excel documents. This feature requires the following third-party libraries:
+
+- <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v4+
+- <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> v2.0.2+
+
+To configure export operations, use the [export{WidgetName}(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#export{WidgetName}options) method from the [excelExporter](/Documentation/ApiReference/Common/Utils/excelExporter/) module. It requires a {WidgetName} instance and a target Excel worksheet.
+
+This method can be called at any point in your application. The following code calls it in the {WidgetName}'s [onExporting](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#onExporting) event handler. In this code, **export{WidgetName}(options)** exports grid data as a <a href="https://en.wikipedia.org/wiki/Binary_large_object" target="_blank">blob</a> that is then saved to an XLSX file. The `e.cancel` parameter disables the deprecated built-in export functionality. The **onExporting** event handler is executed when users click the Export button. The [export](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/export/#enabled) property adds this button to the {WidgetName}.
```

Comment:
> This method can be called at any point in your application.

Avoidable passive voice. We can rewrite this:

> You can call this method at any point in your application.

> This method can be called at any point in **your application**. The following code calls **it** in the {WidgetName}'s [onExporting](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#onExporting) event handler.

Right now "it" points at "your application in the previous sentence but clearly you meant "this method"...

> In this code, **export{WidgetName}(options)** exports grid data as a <a href="https://en.wikipedia.org/wiki/Binary_large_object" target="_blank">blob</a> that **is then saved to** an XLSX file.

Passive voice. Who saves grid data to an XLSX file?

Reply:
> Right now "it" points at "your application in the previous sentence but clearly you meant "this method"...

There's only one thing that's uses the verb "call" in the previous sentence - the method.

> Passive voice. Who saves grid data to an XLSX file?

It is saved using the FileSaver API, but if we say "the FileSaver API saves the grid data to an XLSX file", it sounds weird.

Reply:
I see.